e2e/libs/keywords/pod_keywords.py

```diff
@@ -1,13 +1,15 @@
-import logging
 import time
-
 from node import Nodes
 
 retry_count = 200
 retry_interval = 5
 
 class pod_keywords:
 
+    #TODO
+    # keywords layer can only call lower implementation layer to complete its work
```

Comment:
nit: Can add a readme in keywords folder or add this statement to __init__.py in keywords package.

Reply:
These code is added in https://github.com/longhorn/longhorn-tests/pull/1300. There are still some code in that PR left and need to be refined or removed.